src/loader.js

```diff
@@ -39,7 +39,7 @@ function setupModuleLoader(window) {
      * registered using this mechanism.
      *
      * When passed two or more arguments, a new module is created.  If passed only one argument, an
-     * existing module (the name passed as the first argument to `module`) is retrieved.
+     * existing module of that name is retrieved.
```

Comment:
I like that this is concise --- but how about we make it clear that we're talking about a specific parameter, and maybe just reword the paragraph entirely, because it wasn't particularly well written to begin with. It sounds sort of Yoda-ish, you know? (Referring especially to the preceding "When passed two or more arguments, a new module created is." bit), and just doesn't come across as particularly natural.

Something more like "Creates a new {@link angular.Module} when passed more than one argument, or retrieves an existing {@link angular.Module} when only passed a single argument.", maybe? It's not perfect, but sounds slightly less weird to me.

If we're going to mention "name" at all, lets wrap it in codeblocks (surround with backticks) to hint that it refers to a parameter. And, if we aren't going to mention the position in the list, maybe at least refer to it as "the `name` argument" or something.